TorchSharp/TypeGeneration.tt

```diff
@@ -1086,27 +1129,43 @@ foreach (var type in new [] {
+                new { Name="CDiv", CName= "", ByteResult=false, Doc="element-wise division " },
+                new { Name="CLShift", CName= "", ByteResult=false, Doc="left shift operation of each element" },
+                new { Name="CFMod", CName= "", ByteResult=false, Doc="element-wide remainder of the division (rounded towards zero)" },
+                new { Name="CRemainder", CName= "", ByteResult=false, Doc="element-wise remainder of the division (rounded to nearest)" },
+                new { Name="CBitAnd", CName= "", ByteResult=false, Doc="bitwise and operation of each element " },
+                new { Name="CBitOr", CName= "", ByteResult=false, Doc="bitwise or operation of each element " },
+                new { Name="CBitXor", CName= "", ByteResult=false, Doc="bitwise xor operation of each element " },
+                new { Name="CMax", CName= "", ByteResult=false, Doc="" },
+                new { Name="CMin", CName= "", ByteResult=false, Doc="" },
+                new { Name="LtTensor", CName= "ltTensor", ByteResult=true, Doc="Compare two tensors for '<' and produce a byte tensor with the logical result." },
+                new { Name="LeTensor", CName= "leTensor", ByteResult=true, Doc="Compare two tensors for '<=' and produce a byte tensor with the logical result." },
+                new { Name="GtTensor", CName= "gtTensor", ByteResult=true, Doc="Compare two tensors for '>' and produce a byte tensor with the logical result." },
+                new { Name="GeTensor", CName= "geTensor", ByteResult=true, Doc="Compare two tensors for '>=' and produce a byte tensor with the logical result." },
+                new { Name="EqTensor", CName= "eqTensor", ByteResult=true, Doc="Compare two tensors for equality and produce a byte tensor with the logical result." },
+                new { Name="NeTensor", CName= "neTensor", ByteResult=true, Doc="Compare two tensors and inequality and produce a byte tensor with the logical result." },
+                new { Name="LtTensorT", CName= "ltTensorT", ByteResult=false, Doc="Compare two tensors for '<' and produce a tensor with the logical result." },
+                new { Name="LeTensorT", CName= "leTensorT", ByteResult=false, Doc="Compare two tensors for '<=' and produce a tensor with the logical result." },
+                new { Name="GtTensorT", CName= "gtTensorT", ByteResult=false, Doc="Compare two tensors for '>' and produce a tensor with the logical result." },
+                new { Name="GeTensorT", CName= "geTensorT", ByteResult=false, Doc="Compare two tensors for '>=' and produce a tensor with the logical result." },
+                new { Name="EqTensorT", CName= "eqTensorT", ByteResult=false, Doc="Compare two tensors for equality and produce a tensor with the logical result." },
+                new { Name="NeTensorT", CName= "neTensorT", ByteResult=false, Doc="Compare two tensors and inequality and produce a tensor with the logical result." },
```

Comment:
We should normalize the Doc format (maybe later once we are done with the porting). Some sentences end with period, some with space, some with neither of the two.